src/adr/articles/0001-record-architecture-decisions.md

```diff
@@ -2,7 +2,7 @@
 layout: layouts/adr.njk
 title: "Record architecture decisions"
 weight: 1
-date: 2021-06-28
+date: 2021-06-27
```

Comment:
It doesn't really matter, but this looks like it may be a testing artefact

Reply:
Was a left over amend I had that I snuck in on this, good spot, thank you